Merge master into live-based

OPSTestProduction opened the pull request.

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

jiasli committed:
Add only_show_errors and no_color

e2ebd8 commented:
Docs Build status updates of commit _[9bad822](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/commits/9bad8226ea90367b20da31f3a360490d5cdd6f81)_: 

:clock10: **Full build**: the average full build time is _7 min(s) 14 sec(s)_, this is based on the last _81_ build(s) for this repository.

e2ebd8 commented:
Docs Build status updates of commit _[9bad822](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/commits/9bad8226ea90367b20da31f3a360490d5cdd6f81)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C13%5C0762c6d6-cb91-510a-10b9-a881d0bb3e83%5CPullRequest%5C202004130603529608-95%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=RX9eALNRO4tVYQq1i4FYJ2A4xel18ydBSY720okXgTA%3D&st=2020-04-13T06%3A10%3A36Z&se=2020-05-14T06%3A15%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

azclibot committed:
Update CLI extensions available doc.

Triggered by Azure CLI Extensions Sync Pipeline - ADO_BUILD_URI=356042

https://github.com/Azure/azure-cli-extensions/commit/b56a1faa4c2a3ec290ad7d3a18e40bc33f30efb7

fengzhou-msft committed:
Merge pull request #1930 from azclibot/az-ext-list-ci-356042

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.

Triggered by Azure CLI Extensions Sync Pipeline - ADO_BUILD_URI=357537

https://github.com/Azure/azure-cli-extensions/commit/7862bf03bd6a7aaf1b244327fdbe966bc7e93436

Commit:
Merge pull request #1932 from azclibot/az-ext-list-ci-357537

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.

Triggered by Azure CLI Extensions Sync Pipeline - ADO_BUILD_URI=359076

https://github.com/Azure/azure-cli-extensions/commit/a67b5e09aa6fcfd8c5c809570522cac2b306020c

Commit:
Merge pull request #1933 from azclibot/az-ext-list-ci-359076

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
azure-cli release notes 2.4.0

fengzhou-msft committed:
Apply suggestions from code review

jsntcy committed:
Merge pull request #1937 from azclibot/cli-release-notes-2.4.0

release note for azure cli 2.4.0

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.

Triggered by Azure CLI Extensions Sync Pipeline - ADO_BUILD_URI=364824

https://github.com/Azure/azure-cli-extensions/commit/09eecf8ce0de9ddffd37655f9b3daf58f641d9a4

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
Merge pull request #1942 from azclibot/az-ext-list-ci-364824

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.

Triggered by Azure CLI Extensions Sync Pipeline - ADO_BUILD_URI=370163

https://github.com/Azure/azure-cli-extensions/commit/5e3d07ce2760d25195135e0fac8e08c853ef40d6

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
Merge pull request #1950 from azclibot/az-ext-list-ci-370163

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

dbradish-microsoft committed:
Merge pull request #1924 from jiasli/configuration

Add only_show_errors and no_color

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.

Triggered by Azure CLI Extensions Sync Pipeline - ADO_BUILD_URI=372607

https://github.com/Azure/azure-cli-extensions/commit/3ede273a305e65e1c08ad27b8ff03e4059da32a2

Commit:
Merge pull request #1951 from azclibot/az-ext-list-ci-372607

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
azure-cli release notes 2.5.0

jsntcy committed:
Apply suggestions from code review

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

jsntcy committed:
Merge pull request #1952 from azclibot/cli-release-notes-2.5.0

release note for azure cli 2.5.0

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.

Triggered by Azure CLI Extensions Sync Pipeline - ADO_BUILD_URI=375843

https://github.com/Azure/azure-cli-extensions/commit/71449ed3264ebc45ce12eb3d0e83d999340ffc61

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
azure-cli release notes 2.5.1

jsntcy committed:
Apply suggestions from code review

jsntcy committed:
Merge pull request #1959 from azclibot/cli-release-notes-2.5.1

release note for azure cli 2.5.1

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
Merge pull request #1958 from azclibot/az-ext-list-ci-375843

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.

Triggered by Azure CLI Extensions Sync Pipeline - ADO_BUILD_URI=379330

https://github.com/Azure/azure-cli-extensions/commit/78df589353ca798365e1dc49e91514b1eab2f84f

Commit:
Merge pull request #1963 from azclibot/az-ext-list-ci-379330

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.

Triggered by Azure CLI Extensions Sync Pipeline - ADO_BUILD_URI=383582

https://github.com/Azure/azure-cli-extensions/commit/5926f06d95a17a1b26c84d6f64bdb5c31cf03595

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
Merge pull request #1969 from azclibot/az-ext-list-ci-383582

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)
